dashboard/app/views/devise/sessions/new.html.haml

```diff
@@ -44,7 +45,7 @@
           %button#signup-button.secondary{type: 'button'}= t('nav.user.signup')
 
   .span4
-    .sidebar= render "devise/shared/links"
+    .sidebar= render "devise/shared/links" unless @hide_oauth_sidebar
```

Comment:
@poorvasingal, do you have thoughts on removing the sidebar? That was part of the spec in your e-mail, but I don't know the exact motivation, was it to make sure people in this flow don't try to go back through oauth again?